azure-pipelines.yml

```diff
@@ -0,0 +1,109 @@
+trigger:
+  - master
+
+pr:
+  - master
```

Comment:
I'm guessing the branch name refers to the target branch, not the source branch, or this wouldn't have triggered? I couldn't find it in the docs.

Could add an autoCancel
```yaml
pr:
  branches:
    include: "*"
  autoCancel: true
```

Reply:
Your suggestion appears to be the default value if we don't include a `pr` section at all, so I removed this.